Sources/Sentry/include/SentryHub+Private.h

```diff
@@ -29,6 +29,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, strong) NSMutableArray<id<SentryIntegrationProtocol>> *installedIntegrations;
 
+// Do not use this directly, instead use the non-underscored `logger` property that is
+// defined through a SentryHub.swift file.
+@property (nonatomic, readonly, strong) NSObject *_swiftLogger;
```

Comment:
`l`: Thanks, I'm unsure if the code comment is still 100% correct. It's in the private header now. Maybe we can even ditch the comment.